File: mobileapps/android/apps/MultiTouch/app/src/main/java/com/petamind/multitouch/MainActivity.java
package com.petamind.multitouch;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

/**
 * @author tung.nguyen
 */
public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(new GameScene(this));
    }
}

File: mobileapps/android/games/BasicCollision/app/src/main/java/com/petamind/basiccollision/MainActivity.kt
package com.petamind.basiccollision

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.Menu
import android.view.MenuItem

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(RectVRect(this))
    }

    override fun onCreateOptionsMenu(menu: Menu?): Boolean {
        menuInflater.inflate(R.menu.menu, menu)
        return true
    }

    override fun onOptionsItemSelected(item: MenuItem): Boolean {
        when(item.itemId){
            R.id.rvsp -> setContentView(RectVPoint(this))
            R.id.rvsr -> setContentView(RectVRect(this))
            R.id.rvsc -> setContentView(RectVCircle(this))
        }
        return super.onOptionsItemSelected(item)
    }
}